subsys/partition_manager/Kconfig

```diff
@@ -158,6 +159,17 @@ partition=TFM_EXTRA
 partition-size=0x10000
 rsource "Kconfig.template.partition_size"
 
+if NRF_MODEM_LIB
+
+config TFM_PROFILE
+	string "The build profile used for TFM Secure image."
+	depends on BUILD_WITH_TFM
+	default "profile_small"
+	help
+	  Use profile_small when the modem is enabled to get the RAM usage below
+	  64 kB, so the Modem's IPC RAM area can fit at 0x10000.
+
+endif # TFM_PROFILE
```

Comment:
`endif # NRF_MODEM_LIB`

Could you add a comment here to explain that we are overriding the default value for TFM_PROFILE? The helptext does not hint to this, and it looks like a new declaration.

Reply:
Done